c++/src/kj/filesystem.h

```diff
@@ -157,6 +157,13 @@ class Path {
   bool operator>=(PathPtr other) const;
   // Compare path components lexically.
 
+  bool operator==(const Path& other) const;
+  bool operator!=(const Path& other) const;
+  bool operator< (const Path& other) const;
+  bool operator> (const Path& other) const;
+  bool operator<=(const Path& other) const;
+  bool operator>=(const Path& other) const;
```

Comment:
Could this be simplified with the new spaceship operator? https://devblogs.microsoft.com/cppblog/simplify-your-code-with-rocket-science-c20s-spaceship-operator/

I suppose that would require a new #include, and more #iffery, so might not actually save many lines.

(Side note: I always wanted to be a spaceship operator.)

Reply:
I suppose we could delete everything except `==` and `<` since I think C++ will derive the rest from those.

But I want nothing to do with the spaceship operator. `strcmp()` has always hurt my brain trying to remember which result means what.